scripts/run_vox.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 module load stopos
 
-voxdir=${PBS_O_HOME}/EC14-voxelyze/voxelyzeMain
+voxdir=${PBS_O_HOME}/thesis/voxelyze/voxelyzeMain
```

Comment:
wat? "thesis"?
And also no, changing the name and adding another subfolder fucks with compilation scripts for voxelyze.